src/github/deployment.ts

```diff
@@ -21,7 +21,7 @@ export const deploymentWebhookHandler = async (context: CustomContext, jiraClien
 };
 
 export const processDeployment = async (
-	github: GitHubAPI,
+	_github: GitHubAPI,
```

Comment:
Can't remove this (_github) until all process functions are aligned so have given it the prepended _ for I dont get used!!!